warehouse/models/mart/benefits/_mart_benefits.yml

```diff
@@ -75,31 +49,57 @@ models:
       - name: uuid
         description: UUID
-      - name: sample_rate
-        description: ""
-      - name: insert_id
-        description: string
-      - name: insert_key
-        description: string
-      - name: is_attribution_event
-        description: bool
-      - name: amplitude_attribution_ids
-        description: string
-      - name: partner_id
-        description: string
-      - name: schema
-        description: int
       - name: processed_time
         description: UTC ISO-8601 timestamp
+      - name: event_properties_auth_provider
+        description: The `auth_provider` value from the `event_properties` column
+      - name: event_properties_card_tokenize_func
+        description: The `card_tokenize_func` value from the `event_properties` column
+      - name: event_properties_card_tokenize_url
+        description: The `card_tokenize_url` value from the `event_properties` column
+      - name: event_properties_eligibility_verifier
+        description: The `eligibility_verifier` value from the `event_properties` column
+      - name: event_properties_error_name
+        description: The `error` value from the `event_properties` column
+      - name: event_properties_error_status
+        description: The `error` value from the `event_properties` column
+      - name: event_properties_error_sub
+        description: The `error` value from the `event_properties` column
+      - name: event_properties_href
+        description: The `href` value from the `event_properties` column
+      - name: event_properties_language
+        description: The `language` value from the `event_properties` column
+      - name: event_properties_origin
+        description: The `origin` value from the `event_properties` column
+      - name: event_properties_path
+        description: The `path` value from the `event_properties` column
+      - name: event_properties_payment_group
+        description: The `payment_group` value from the `event_properties` column
+      - name: event_properties_status
+        description: The `status` value from the `event_properties` column
+      - name: event_properties_transit_agency
+        description: The `transit_agency` value from the `event_properties` column
+      - name: event_properties_eligibility_types
+        description: A semi-colon delimited list of `eligibility_types` values from the `event_properties` column
+      - name: user_properties_eligibility_verifier
+        description: The `eligibility_verifier` value from the `user_properties` column
+      - name: user_properties_initial_referrer
+        description: The `initial_referrer` value from the `user_properties` column
+      - name: user_properties_initial_referring_domain
+        description: The `initial_referring_domain` value from the `user_properties` column
+      - name: user_properties_provider_name
+        description: The `provider_name` value from the `user_properties` column
```

Comment:
Sorry, didn't notice this one until now -- it's the older version of `transit_agency`, so let's drop it.

Reply:
Just to confirm. `provider_name` should be dropped because it was superseded by `transit_agency`. Correct?

Reply:
Yep we renamed it in https://github.com/cal-itp/benefits/pull/901

Reply:
I added a `COALESCE` to pick data from `provider_name` if it exists (for any entries predating that PR being merged); see 38218bbb2775e57709f4186e4589d22c3b614527

Reply:
Even better!